site/docs/v1/tech/release-notes.adoc

```diff
@@ -49,21 +49,68 @@ include::docs/v1/tech/__database-migration-warning.adoc[]
 
 === New
 * JWT Vending machine
-* IP Access Control. Requires Threat Detection feature.
+** This allows a JWT to be created for a not-yet-existing user with a payload defined by the API caller.
+** Resolves https://github.com/FusionAuth/fusionauth-issues/issues/525[GitHub Issue #525]
+* FusionAuth wasn't awesome enough, so we added a robust Threat Detection feature for enterprise customers. This feature includes: 
+* IP Access Control for API keys
+** This allows domains or API addresses to be safelisted to use an API key.
```

Comment:
I would have to check, but I think this is a block by IP address for an API key.  Not 100 on this one; I would get confirmation during standup